include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -1,7 +1,6 @@
 #pragma once
 
-#include <optional>
-
+#include "can/core/can_writer_task.hpp"
```

Comment:
Mistake?

Reply:
I don't think so. I believe that it solved a build error.